src/rf/apps/workers/status_updates.py

```diff
@@ -132,6 +132,24 @@ def mark_image_transfer_failure(s3_uuid, error_message):
                                  ERROR_MESSAGE_LAYER_IMAGE_TRANSFER)
 
 
+def update_thumbnail_status(layer_id):
```

Comment:
The name of this function isn't very clear. It could be updating data on the layer or the image itself. I think it is worth renaming to make the expected functionality more explicit.